swap/cashout.go

```diff
@@ -34,18 +35,21 @@ const CashChequeBeneficiaryTransactionCost = 50000
 type CashoutProcessor struct {
 	backend    chain.Backend     // ethereum backend to use
 	privateKey *ecdsa.PrivateKey // private key to use
+	Logger     Logger
```

Comment:
Is there a reason why the `Logger` field is always exported?

Reply:
Good question, This was exported to be used from outside the swap package, this can be revised when we refactor into a package.